### ISSUE TITLE
JSEP mentions non-existent "TCP/TLS/RTP/SAVPF"

### ISSUE THREAD

**Issue by adamroach** [rtcweb-wg/jsep#860]
§5.1.2 says:

```
   For instance, an endpoint might generate "RTP/AVP"
   but supply "a=fingerprint" and "a=rtcp-fb" attributes, indicating its
   willingness to support "UDP/TLS/RTP/SAVPF" or "TCP/TLS/RTP/SAVPF".
```

The string `TCP/TLS/RTP/SAVPF` is not registered in [the IANA SDP proto registry](https://www.iana.org/assignments/sdp-parameters/sdp-parameters.xhtml#sdp-parameters-2). Presumably, this text should say `TCP/DTLS/RTP/SAVPF`.

**Comment by adamroach** [comment]
Opened PR #861.

**Comment by adamroach** [comment]
Fixed in #861
